Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.ts

```diff
@@ -1,19 +1,46 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core'
+import {
+  FormGroup,
+  FormControl,
+  Validators,
+} from "@angular/forms";
```

Comment:
This can all be on a single line and the comma following Validators can be removed.